cdc/cdc.cc

```diff
@@ -357,13 +386,23 @@ class transformer final {
 
                         values[0] = data_type_for<column_op_native_type>()->decompose(data_value(static_cast<column_op_native_type>(op)));
                         res.set_cell(log_ck, *dst, atomic_cell::make_live(*dst->type, _time.timestamp(), tuple_type_impl::build_value(values)));
+
+                        if (pirow && pirow->has(cdef.name_as_text())) {
+                            values[0] = data_type_for<column_op_native_type>()->decompose(data_value(static_cast<column_op_native_type>(column_op::set)));
+                            values[1] = pirow->get_blob(cdef.name_as_text());
+                            values[2] = std::nullopt;
```

Comment:
And a FIXME for TTL.

Reply:
Again, we don't get a ttl back. This is a snapshot of how this looks right now, by value. We have data bytes, that is is. I think that is fine.